PyDynamic/uncertainty/interpolate.py

```diff
@@ -169,8 +169,13 @@ def interp1d_unc(
 
         if fill_unc == "extrapolate":
             fill_unc = uy[0], uy[-1]
-        elif returnC:
-            # Once we deal with this, we will probably introduce another input parameter
+        elif bounds_error is not None and returnC:
```

Comment:
If we use an `elif` here, this case gets only checked if `fill_unc != "extrapolate"`. Is this the intended behavior?

Reply:
TL;DR
Yes, it is.

Check to additionally added comment in the following lines for that.